packages/uniforms-bridge-json-schema/src/JSONSchemaBridge.ts

```diff
@@ -108,30 +126,40 @@ export default class JSONSchemaBridge extends Bridge {
     const rootName = joinName(nameParts.slice(0, -1));
     const baseName = nameParts[nameParts.length - 1];
     const scopedError = details.find(error => {
-      const path = pathToName(error.instancePath ?? error.dataPath);
+      const path = pathToName((error.instancePath ?? error.dataPath) || '');
       return (
         unescapedName === path ||
-        (rootName === path && baseName === error.params.missingProperty)
+        (rootName === path &&
+          error.params &&
+          baseName === error.params.missingProperty)
```

Comment:
```suggestion
          error.params &&
          baseName === error.params?.missingProperty)
```

Reply:
Actually, that one is not necessary because we check for `error.params` truthiness.
On the other hand, if I removed this check and replaced with `?.`, the coverage went down a little because `?.` branch was not covered. I found it difficult to cover, so I just left it.